### PR TITLE
FAT-944 - Negative scenario - fail to update Holdings, and Items

### PR DESCRIPTION
**Pull request by SatishkumarPurswani**
'Match MARC-to-MARC and update Instances, fail to update Holdings, and Items' scenario - INTEGRATION

~_Execution of these test cases is pending partially._~
**Update**:  Test cases were executed pointing to snapshot-2 environment

### CODE REVIEW

File: data-import/src/main/resources/folijet/data-import/features/data-import-integration.feature [hunk 1]
```diff
@@ -5060,10 +5060,1449 @@ Feature: Data Import integration tests
...
+        "name": "FAT-944_New: MARC-to-Instance",
+        "incomingRecordType": "MARC_BIBLIOGRAPHIC",
+        "existingRecordType": "INSTANCE",
+        "description": "FAT-944_New: MARC-to-Instance",
+        "mappingDetails": {
+          "name": "instance",
+          "recordType": "INSTANCE",
+          "mappingFields": [
+            {
+              "name": "discoverySuppress",
```
Comment (Abdulkhakimov) [inline data-import/src/main/resources/folijet/data-import/features/data-import-integration.feature:5083]: A lot of empty elements from mappingFields have no effect when saving the profile. You can delete them.

Comment (SatishkumarPurswani) [reply data-import/src/main/resources/folijet/data-import/features/data-import-integration.feature:5083]: @Abdulkhakimov  - I'll take care about these in future PR. Hope that is fine?

File: data-import/src/main/resources/folijet/data-import/features/data-import-integration.feature [hunk 2]
```diff
@@ -5060,10 +5060,1449 @@ Feature: Data Import integration tests
...
+        "dataType": "MARC"
+      }
+    }
+    """
+    When method POST
+    Then status 204
+
+    ## verify job execution for data-import
+    * call pause 180000
+    * call read('classpath:folijet/data-import/features/get-completed-job-execution.feature@getJobWhenJobStatusCompleted') { jobExecutionId: '#(importJobExecutionId)'}
```
[the remark this answers is not in the document]
Comment (SatishkumarPurswani) [reply data-import/src/main/resources/folijet/data-import/features/data-import-integration.feature:6497]: @KaterynaSenchenko - Have added the call as mentioned by you.

File: data-import/src/main/resources/folijet/data-import/features/data-import-integration.feature [hunk 3]
```diff
@@ -5060,11 +5060,1488 @@ Feature: Data Import integration tests
...
+    And match jobExecution.progress == '#present'
+
+    # verify that needed entities created
+    * call pause 10000
+    Given path 'metadata-provider/jobLogEntries', importJobExecutionId
+    And headers headersUser
+    When method GET
+    Then status 200
+    And assert response.entries[0].sourceRecordActionStatus == 'CREATED'
+    And assert response.entries[0].instanceActionStatus == 'CREATED'
```
Comment (KaterynaSenchenko) [inline data-import/src/main/resources/folijet/data-import/features/data-import-integration.feature:6513]: Shouldn't the status be UPDATED for the Instance and DISCARDED for Holdings and Item?

**Pull request merged**